src/app/beer_garden/events/handlers.py

```diff
@@ -33,7 +34,7 @@ def garden_callbacks(event: Event) -> None:
 
     # These are all the MAIN PROCESS subsystems that care about events
     for handler in [
-        beer_garden.application.handle_event,
+        #beer_garden.application.handle_event,
```

Comment:
I'm being pedantic, so sorry, but I think that commented out code should be deleted in nearly every case. We can find the history through git if we need to see how things used to be done.

Reply:
I actually wasn't removing it, as it's likely still required. However, there is a weird issue that prevents patching this module:

`beer_garden.application` isn't actually a module, it's instantiated in `beer_garden.__main__`. When `handlers.py` runs, one of the imports puts it in the namespace. However, this is *not* instantiated when running a test. I'm still trying to figure out an elegant way to get around that (it's stumped me for a bit).